src/language_features/completion.rs

```diff
@@ -70,17 +69,14 @@ pub fn editor_completion(
                 .and_then(|compl| compl.resolve_provider)
                 .unwrap_or(false)
             {
-                format!(
-                    "set-option window lsp_completions_selected_item {}; \
-                        lsp-completion-item-resolve-request true; ",
-                    completion_item_index
-                )
+                "lsp-completion-item-resolve-request true".to_string()
             } else {
                 "".to_string()
             };
-            let on_select = format!(
-                "{}info -markup -style menu -- %§{}§",
-                &maybe_set_index,
+            let on_select = formatdoc!(
+                "lsp-completion-item-selected {completion_item_index}
+                {maybe_set_index} 
```

Comment:
now that this variable no longer sets the index, it should be renamed

also, it's our convention to align the line contents, so the "l" should align with the "{".

```suggestion
                 {maybe_resolve} 
```